connector/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -664,8 +664,10 @@ public long getTableModificationTime(ConnectorSession session, ConnectorTableHan
             // since HDFS is append-only and any table modification will trigger directory update.
```

Comment:
i try to build omnidata-openlookeng-connector, and found HiveMetadata.java updated in openLooKeng, so our compile will fail, pls rebase the new code.
line 3133 canPerformSortBasedAggregation change to validateAndGetSortAggregationType, pls check.